generators/generator-base-private.js

```diff
@@ -339,21 +339,31 @@ module.exports = class extends Generator {
      * @returns formatted api description
      */
     formatAsApiDescription(text) {
+        if (!text || text === '') {
+            return text;
+        }
         const rows = text.split('\n');
         let description = rows[0];
         for (let i = 1; i < rows.length; i++) {
             // discard empty rows
-            if (rows[i] !== '') {
+            if (rows[i] !== '' && rows[i].trim() !== '') {
```

Comment:
can be just `rows[i].trim() !== ''` as it will cover the first case as well

Reply:
true that